modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java

```diff
@@ -523,18 +512,14 @@ private String _getPortalCacheKey(
 		String ddmDataProviderId, List<KeyValuePair> keyValuePairs,
 		String url) {
 
-		Stream<KeyValuePair> stream = keyValuePairs.stream();
-
 		return StringBundler.concat(
 			ddmDataProviderId, StringPool.AT, url, StringPool.QUESTION,
-			stream.sorted(
-			).map(
+			com.liferay.petra.string.StringUtil.merge(
```

Comment:
@W2jx I missed this one in last review. Please make it a for-loop with StringBundler.